inc/views/nav_walker.php

```diff
@@ -172,14 +179,17 @@ private function get_caret_pictogram( $settings ) {
 	}
 
 	/**
-	 * Get mobile submenu inline styles
+	 * Get mobile submenu inline styles and accessibility
 	 */
-	public function get_mobile_submenu_style() {
+	public function get_mobile_submenu_and_accessibility_style() {
 		$mobile_button_caret_css  = '.header-menu-sidebar .nav-ul li .wrap { position:relative; padding: 15px 0; display: flex; align-items: center; }';
 		$mobile_button_caret_css .= '.header-menu-sidebar .nav-ul li .wrap a { flex-grow: 1; }';
 		$mobile_button_caret_css .= '.header-menu-sidebar .nav-ul li .wrap a .dd-title { width: var(--wrapdropdownwidth); }';
 		$mobile_button_caret_css .= '.header-menu-sidebar .nav-ul li .wrap button { border: 0; z-index: 1; background: 0; }';
-		return Dynamic_Css::minify_css( $mobile_button_caret_css );
+		$accessibility_caret_css  = '.nav-ul:not(.menu-mobile) li { display: flex; align-items: center; }';
```

Comment:
@preda-bogdan I am pretty sure this line here will break both the free & pro **mega-menu** features.

We should make sure that it still works fine even after the changes.

Reply:
@abaicus thank you for mentioning it, I managed to find a solution that works on both. Let me know what you think.